tests/series_only/is_between_test.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pandas as pd
+import pytest
+
+import narwhals.stable.v1 as nw
+from narwhals.utils import parse_version
+from tests.utils import compare_dicts
+
+data = [1, 4, 2, 5]
+
+
+@pytest.mark.parametrize(
+    ("closed", "expected"),
+    [
+        ("left", [True, True, True, False]),
+        ("right", [False, True, True, True]),
+        ("both", [True, True, True, True]),
+        ("none", [False, True, True, False]),
+    ],
+)
+def test_is_between(
+    request: Any, constructor_series: Any, closed: str, expected: list[bool]
+) -> None:
+    if "pandas_series_nullable_constructor" in str(constructor_series) and parse_version(
+        pd.__version__
+    ) < (2, 2):
+        # bug in old pandas
+        request.applymarker(pytest.mark.xfail)
```

Comment:
I think you can just remove this - I suspect you got it from the `value_counts` tests, but that was for a very specific bug in pandas for `value_counts`. in general, we don't need it

Reply:
yes, excatly it was from value_counts and I didn't understand its purpose that's why I asked if I should leave it there.

Reply:
ah I see sorry, I thought you were asking something like "it's already in an existing `test_is_between` test, should I leave it there?" - all good then 🙌